### ISSUE TITLE
Missing some localized messages in security screens

### ISSUE THREAD

**Issue by pfurini** [cuba-platform/translations#45]
Hi, I'm testing my soon to be published add-on for Italian localization, and I noticed some missing translations, that are indeed present in my properties files.

Here is what I found until now:

![screen shot 2018-12-03 at 17 51 16](https://user-images.githubusercontent.com/4931051/49388369-19e32500-f724-11e8-94c2-0b9baf97a5b2.png)
![screen shot 2018-12-03 at 17 49 03](https://user-images.githubusercontent.com/4931051/49388381-1fd90600-f724-11e8-9ae8-0b586cec0f76.png)

the affected messages are in the `gui` module, and some of them are:
```
com.haulmont.cuba.gui.app.security.group/action.import.caption = Importa JSON o ZIP
com.haulmont.cuba.gui.app.security.group/activate = Attiva
com.haulmont.cuba.gui.app.security.group/create.caption = Crea
com.haulmont.cuba.gui.app.security.group/exportJSON = Esporta come JSON
com.haulmont.cuba.gui.app.security.group/exportZIP = Esporta come ZIP
```

I don't know why some of them are applied (like the others from `com.haulmont.cuba.gui.app.security.group/` namespace) while others are not.

Please let me know if there's something I can do to fix them on my side, thanks.

Paolo

**Comment by knstvk** [comment]
Hi Paolo,
It can be caused by the fact that new projects are now created without `gui` module, so when applying a translation, you have to copy contents of the translation `gui` module to project's `web` module. I've explained it in the translations README.
Please try to do it in your component, so you'll just have 2 copies of the same file in `gui` and `web`.

**Comment by pfurini** [comment]
@knstvk you're right, that might be the issue, I didn't remember that fact.. I'll try ASAP your suggestion and report back here.

P.

**Comment by pfurini** [comment]
Hi @knstvk, unfortunately it doesn't work even by copying `gui` messages under the `web` module, as follows:

<img width="303" alt="screen shot 2018-12-12 at 12 13 57" src="https://user-images.githubusercontent.com/4931051/49866294-a4541480-fe07-11e8-855d-246addf811c2.png">

I just uploaded an updated version of my `translation-it` component (6.10.1) to bintray, and you can find the sources [here](https://github.com/pfurini/cuba-translation-it).
After adding it to a project the outcome is the same as I posted above.

P.

**Comment by knstvk** [comment]
Some message packs in the framework were not located properly for localization. So I had to duplicate some messages in translations, now it should be ok. 
Will fix the location in the framework for 7.0.

**Comment by pfurini** [comment]
Thanks @knstvk, that was it! I just updated my translation add-on and I'm going to upload it on bintray.

P.
